client/post-editor/editor-action-bar/index.jsx

```diff
@@ -67,7 +67,7 @@ export default React.createClass( {
 
 		return (
 			<div className="editor-action-bar">
-				<div className="editor-action-bar__first-group">
+				<div className="editor-action-bar__cell editor-action-bar__cell-left">
```

Comment:
I wasnt' sure about the modifiers, but I will fix them to follow the coding guidelines for CSS. 😄